pom.xml

```diff
@@ -0,0 +1,265 @@
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.vlkan</groupId>
+    <artifactId>reactor-pubsub</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <name>Reactive Google Cloud Pub/Sub driver with Reactor</name>
+    <description>
+        Finally a Google Cloud Pub/Sub driver that you can wrap your mind around its internals and leverage this to
+        implement efficient, backpressure-aware, asynchronous, debugging-friendly Pubsub requests that play nicely with
+        your existing Reactor code base.
+    </description>
+    <url>https://github.com/vy/reactor-pubsub</url>
+
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <id>vy</id>
+            <name>Volkan Yazıcı</name>
+            <email>volkan.yazici@gmail.com</email>
+            <organization>Volkan Yazıcı's Soap Co.</organization>
+            <organizationUrl>https://vlkan.com/</organizationUrl>
+            <roles>
+                <role>Project-Administrator</role>
+                <role>Developer</role>
+            </roles>
+            <timezone>UTC+01:00</timezone>
+        </developer>
+    </developers>
+
+    <scm>
+        <url>http://github.com/vy/reactor-pubsub</url>
+        <connection>scm:git:git@github.com:vy/reactor-pubsub.git</connection>
+        <developerConnection>scm:git:git@github.com:vy/reactor-pubsub.git</developerConnection>
+        <tag>HEAD</tag>
+    </scm>
+
+    <distributionManagement>
+        <snapshotRepository>
+            <id>sonatype-nexus-snapshots</id>
+            <name>Sonatype Nexus snapshot repository</name>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+        </snapshotRepository>
+        <repository>
+            <id>sonatype-nexus-staging</id>
+            <name>Sonatype Nexus release repository</name>
+            <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+        </repository>
+    </distributionManagement>
+
+    <properties>
+
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <java.version>1.8</java.version>
+
+        <!-- library versions -->
+        <assertj-core.version>3.12.2</assertj-core.version>
+        <blockhound.version>1.0.0.M4</blockhound.version>
+        <jackson.version>2.9.9</jackson.version>
+        <jsr305.version>3.0.2</jsr305.version>
+        <junit.version>4.13-beta-3</junit.version>
+        <google-auth-library-oauth2-http.version>0.17.1</google-auth-library-oauth2-http.version>
+        <log4j.version>2.11.2</log4j.version>
+        <micrometer.version>1.2.0</micrometer.version>
+        <reactor-core.version>3.3.0.M2</reactor-core.version>
```

Comment:
consider using the BOM, to have -core and -netty aligned. Be aware that RC1 is out now 🎉

Reply:
I am not a Maven expert, but this is what I ended up with:

1. Added the following snippet into `pom.xml`:
```xml
<dependencyManagement>
    <dependencies>
        <dependency>
            <groupId>io.projectreactor</groupId>
            <artifactId>reactor-bom</artifactId>
            <version>Bismuth-RELEASE</version>
            <type>pom</type>
            <scope>import</scope>
        </dependency>
    </dependencies>
</dependencyManagement>
```
2. Removed `<version>` tags from `reactor-core` and `reactor-netty` dependencies.
3. Kicked `mvn compile` and had the following error:
```
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] 'dependencies.dependency.version' for io.projectreactor.netty:reactor-netty:jar is missing. @ line 125, column 21
```

Any ideas?

Reply:
try `Dysprosium-RELEASE` (released last week)